include/mysql.h

```diff
@@ -227,6 +227,8 @@ extern const char *SQLSTATE_UNKNOWN;
     /* MariaDB specific */
     MYSQL_PROGRESS_CALLBACK=5999,
     MYSQL_OPT_NONBLOCK,
+    MYSQL_OPT_FOLLOW_SERVER_REDIRECTS,
```

Comment:
Should be prefixed with MARIADB unless MySQL/Oracle has the same option.

Reply:
Okay, will do…

But why are the immediately preceding options _both_ prefixed with `MYSQL` _and_ marked as `/* MariaDB specific */?

Those are contradictory, are they not?